### PR TITLE
MANAGER-3457

### PR DESCRIPTION
**Pull request by euhmeuh**
Ref: MANAGER-3457

### CODE REVIEW

File: client/app/billing/confirmTerminate/billing-confirmTerminateLegacy.service.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
+export default class {
+  constructor($q, OvhHttp) {
+    this.$q = $q;
+    this.OvhHttp = OvhHttp;
+  }
+
+  static getServiceTypeFromPrefix(serviceApiPrefix) {
+    return serviceApiPrefix
```
Comment (FredericEspiau) [inline client/app/billing/confirmTerminate/billing-confirmTerminateLegacy.service.js]: Could you please provide comments about what this does ?

```js
// Turns "hello you" to "h3ll0 y0u"
```

so we can understand easier what it does ? :)

**Comment by ovh-cds** [comment]
CDS Report build-manager#2919.0 ✘  
* Stage 1
  * Build dependencies and inject Sentry ✔ 
* Stage 2
  * Build manager CA ✘  
  * Build manager EU ✘  
  * Build manager US ✘  
  * Tests ✔

**Comment by ovh-cds** [comment]
CDS Report build-manager#2947.0 ✘  
* Stage 1
  * Build dependencies and inject Sentry ✔ 
* Stage 2
  * Build manager CA ✘  
  * Build manager EU ✘  
  * Build manager US ✘  
  * Tests ✔

**Comment by antleblanc** [comment]
Please refer to https://github.com/ovh/manager

**Pull request closed without merging**